pkgs/by-name/de/deltachat-tauri/package.nix

```
@@ -0,0 +1,93 @@
+{
+  cargo-tauri,
+  fetchFromGitHub,
+  lib,
+  libayatana-appindicator,
+  makeWrapper,
+  nodejs,
+  openssl,
+  perl,
+  pkg-config,
+  pnpm_9,
+  python3,
+  rustPlatform,
+  stdenv,
+  versionCheckHook,
+  webkitgtk_4_1,
+  wrapGAppsHook4,
+}:
+
+let
+  pnpm = pnpm_9;
+in
+rustPlatform.buildRustPackage rec {
+  pname = "deltachat-tauri";
+  version = "1.57.0";
+
+  src = fetchFromGitHub {
+    owner = "deltachat";
+    repo = "deltachat-desktop";
+    tag = "v${version}";
+    hash = "sha256-B7rM2NIbkQp9PvpLUYJ8wTKHXOijUZmZ2eyhO4oBNj8=";
+  };
+
+  pnpmDeps = pnpm.fetchDeps {
+    inherit pname version src;
+    hash = "sha256-6UgDT1SK4cVY6nZG9x+YBs1YV3tifKwis7cvDx+Ppv8=";
+  };
+
+  useFetchCargoVendor = true;
+  cargoHash = "sha256-YK4YCOmV5cbk871RlG47UJ8BMMy1h6ClRa7IwWlYxUk=";
+
+  postPatch = lib.optionalString stdenv.hostPlatform.isLinux ''
+    substituteInPlace $cargoDepsCopy/libappindicator-sys-*/src/lib.rs \
+      --replace-fail libayatana-appindicator3.so.1 '${libayatana-appindicator}/lib/libayatana-appindicator3.so.1'
+  '';
+
+  nativeBuildInputs =
+    [
+      cargo-tauri.hook
+      nodejs
+      perl
+      pnpm.configHook
+    ]
+    ++ lib.optionals stdenv.hostPlatform.isLinux [
+      pkg-config
+      python3
+      wrapGAppsHook4
+    ];
+
+  buildInputs = lib.optionals stdenv.hostPlatform.isLinux [
+    libayatana-appindicator
+    openssl
+    webkitgtk_4_1
```

```suggestion
    webkitgtk_4_1
    gst_all_1.gstreamer
    gst_all_1.gst-plugins-base
    gst_all_1.gst-plugins-good
    gst_all_1.gst-plugins-bad
    gst_all_1.gst-libav
    gst_all_1.gst-vaapi
```

those seem to be also needed

---

What's the error message you get without these?